bane-of-wargs.html

```diff
@@ -19,6 +19,9 @@
 	<p class="dialog">cd Bane-Of-Wargs/</p>
 	<p class="dialog">pip install -r requirements.txt</p>
 	<p class="dialog">python source/main.py</p>
+      <p></p>
+      <p>Here's the plot of the vanilla data:</p>
+	<p class="dialog">“ You're a lumberjack who lives in a small village in the Forlindon Woods, a quiet a calm place to live. You've always wondered what's after this quiet forest; because your little village is a small town that lives apart from the other great cities in the Stall Insland. You've heard some stories and tales about the rest of the world. Only some words come to your mind when you hear "the world": dragons, gold, elves, dwarves, great mountains, adventure... A day like others, you get a message in your mailbox: a dwarf king from the Goro Mountains Peaks has heard from the mayor of the village, who keeps contact with him because they provide the village in stone, that you have great woodcraft skills. This dwarf king requests you to come to his castle in the mountains of the Goro Peaks to participate to a great project... You decide to accept the deal, because of the great promiced remuneration. You'll travel into many places like the Californ Woods, the Stall Fields, and many other great places. ”</p>
```

Comment:
```suggestion
	<p class="dialog">“ You're a lumberjack who lives in a small village in the Forlindon Woods, a quiet a calm place to live. You've always wondered what's after this quiet forest because your little village is a small town that lives apart from the other great cities in the Stall Insland. You've heard some stories and tales about the rest of the world. Only some words come to your mind when you hear "the world": dragons, gold, elves, dwarves, great mountains, adventure... One day that starts like any other, you get a message in your mailbox: a dwarf king from the Goro Mountains Peaks has heard from the mayor of the village, who keeps contact with him because they provide the village in stone, that you have great woodcraft skills. This dwarf king requests you to come to his castle in the mountains of the Goro Peaks to participate to a great project... You decide to accept the deal, because of the great promiced remuneration. You'll travel into many places like the Californ Woods, the Stall Fields, and many other great places. ”</p>
```